x-pack/test/functional/apps/machine_learning/anomaly_detection/advanced_job.ts

```diff
@@ -278,6 +278,7 @@ export default function({ getService }: FtrProviderContext) {
     this.tags(['smoke', 'mlqa']);
     before(async () => {
       await esArchiver.load('ml/ecommerce');
+      await ml.api.createCalendar('wizard-test-calendar');
     });
 
     after(async () => {
```

Comment:
Calendar cleanup (deletion) should be added to the `after` method.

Reply:
Yeah, I added it first, you can find a method `deleteCalendar`, but there is already a clean-up that happens for ml indices.

Reply:
Dedicated cleanup is not needed (it's done with the `.ml-*` indices cleanup).